teambot.py

```diff
@@ -7,17 +7,21 @@
 import atexit
 
 # This directory maps slack channel IDs (not names) to sets of user IDs
-directory = shelve.open('teams.db')
+directory = None
 
 outputs = []
 my_user_id = None
 slack_client = None
 
-def setup(bot):
+def setup(bot, config):
     global my_user_id
     global slack_client
-    slack_client = bot.slack_client
+    global directory
+
+    dir_file = config.get('TEAM_DB_FILE') or 'teams.db'
```

Comment:
[shelve.open](https://docs.python.org/2/library/shelve.html#shelve.open) adds an extension for me, creating `teams.db.db`.